sw/device/lib/dif/dif_edn.h

```diff
@@ -223,6 +223,18 @@ dif_result_t dif_edn_lock(const dif_edn_t *edn);
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_edn_is_locked(const dif_edn_t *edn, bool *is_locked);
 
+/**
```

Comment:
oops forgot to remove.
didn't realize at the time `dif_edn_stop` already served this function.